helm-chart/renku-notebooks/values.yaml

```diff
@@ -65,3 +59,97 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+## Configuration for the jupyterhub service
+jupyterhub:
+  rbac:
+    enabled: true
+  hub:
+    image:
+      name: renku/jupyterhub-k8s
+      tag: '0.0-unclean-fd4775a'
+    services:
+      notebooks:
+        url: http://renku-notebooks
+        admin: true
+        oauth_client_id: service-notebooks
+        # need to set the api_token
+        # api_token: notebookstoken
+    extraEnv:
+      DEBUG: 1
+      JUPYTERHUB_SPAWNER_CLASS: spawners.RenkuKubeSpawner
+      # GITLAB_URL: http://gitlab.com
+    extraConfig: |
+      import os
+
+      # Set the log level by value or name.
+      c.JupyterHub.log_level = 'DEBUG'
+
+      # Enable debug-logging of the single-user server
+      c.Spawner.debug = True
+
+      # JupyterHub uses GITLAB_HOST which is a misnomer -- we use GITLAB_URL
+      # everywhere so set this mapping here
+      os.environ['GITLAB_HOST'] = os.getenv('GITLAB_URL')
+
+      #: Automatically begin the login process without showing the button.
+      c.Authenticator.auto_login = True
+
+      #: Enable named-servers
+      c.JupyterHub.allow_named_servers = True
+
+      #: Configure the notebook spawner.
+      c.JupyterHub.spawner_class = os.getenv('JUPYTERHUB_SPAWNER_CLASS',
+                                           'spawners.Spawner')
+
+      c.RenkuKubeSpawner.pod_name_template = 'jupyter-{username}{servername}'
+
+      #: Explicitly set the notebook directory because we'll be mounting a host volume to
+      #: it.  Most jupyter/docker-stacks *-notebook images run the Notebook server as
+      #: user `jovyan`, and set the notebook directory to `/home/jovyan/work`.
+      #: We follow the same convention. Note that in most cases this is
+      #: overriden in the RenkuKubeSpawner.
+      notebook_dir = os.getenv('JUPYTERHUB_NOTEBOOK_DIR', '/home/jovyan/work')
+      c.Spawner.notebook_dir = notebook_dir
+
+      #: For debugging arguments passed to spawned containers
+      c.Spawner.debug = bool(os.getenv('DEBUG', False))
+
+      # Increase pod initialization timeout to 15 minutes
+      c.KubeSpawner.start_timeout = 60 * 15
+
+      # prevent redirect to /hub if the server is taking slightly longer to start
+      c.JupyterHub.tornado_settings = {
+        'slow_spawn_timeout': 1
+      }
+
+  singleuser:
+    image:
+      name: renku/singleuser
+      tag: '0.0-unclean-fd4775a'
+    storage:
+      type: none
+    defaultUrl: /lab
+  # FIXME: bug in prepuller makes helm hang in certain cases. Fixed in 0.7
+  # so this should be removed eventually.
+  # https://github.com/jupyterhub/zero-to-jupyterhub-k8s/issues/477
+  prePuller:
+    hook:
+      enabled: false
+  auth:
+    type: gitlab
+    gitlab:
+      # Fill in the clientId and clientSecret after creating the OAuth app
+      # clientId: 1234
+      # clientSecret: 1234
+      callbackUrl: '""' # Forces the default callback url
```

Comment:
clientid is missing, compared to values from https://github.com/SwissDataScienceCenter/renku